app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -840,3 +870,122 @@ class GithubWebhookSubscription extends SubscriptionHandler {
     return true;
   }
 }
+
+typedef PullRequestLabelProcessorProvider = PullRequestLabelProcessor Function({
+  required Config config,
+  required GithubService githubService,
+  required PullRequest pullRequest,
+});
+
+class PullRequestLabelProcessor {
```

Comment:
I realize this class is intended to be package-private, but it would be nice to hide (make private) the methods and members private (`_private`) so that it's more clear what is intended to be used and what isn't (like, I wouldn't want to be calling `processEmergencyLabel()` instead of `processLabels()`)